<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Charging_monitoring</class>
 <widget class="QWidget" name="Charging_monitoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="charge_monitoring">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/E1new.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="full_charge">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/new8dianci.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="full_charge_but">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>128</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../img.qrc">
     <normaloff>:/img/images/new8dianci.png</normaloff>:/img/images/new8dianci.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>128</width>
     <height>64</height>
    </size>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="charge_7but">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>128</width>
     <height>58</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/new7dianci.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="charge_6but">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>128</width>
     <height>58</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/new6dianci.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="charge_5but">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>128</width>
     <height>58</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/new5dianci.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="charge_4but">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>128</width>
     <height>58</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/new4dianci.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="charge_3but">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>128</width>
     <height>58</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/new3dianci.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="charge_2but">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>128</width>
     <height>58</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/new2dianci.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="charge_1but">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>128</width>
     <height>58</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/new1dianci.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_soc">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>156</y>
     <width>126</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="soc_set">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_moni_but">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>98</y>
     <width>98</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="change_equ_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>98</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="change_carinf_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>98</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="change_end_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>98</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_current">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_power">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_vol">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>381</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_status">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>381</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_total">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>255</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_supend">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>295</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_time">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_timelife">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>381</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../img.qrc"/>
 </resources>
 <connections/>
</ui>
